### PR TITLE
Styling improvements for scale-intro.md

### PR DESCRIPTION
**Pull request by Gauravpadam**
follow-up for #44752

 A few layout improvements for the page:
 - The two-column layout is symmetric now
 - The newly added note is on a separate row which adds proper separation

[Original page](https://kubernetes.io/docs/tutorials/kubernetes-basics/scale/scale-intro/)

[Deploy Preview](https://deploy-preview-44767--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/scale/scale-intro/)

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/area web-development

_(Improving the layout through HTML elements restructuring gives off a web development vibe)_

File: content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 1]
```diff
@@ -17,23 +17,19 @@
...
+              <ul>
+                  <li>Scale an app using kubectl.</li>
+              </ul>
+
        <h3>Scaling an application</h3>
 
             <p>Previously we created a <a href="/docs/concepts/workloads/controllers/deployment/"> Deployment</a>, and then exposed it publicly via a <a href="/docs/concepts/services-networking/service/">Service</a>. The Deployment created only one Pod for running our application.  When traffic increases, we will need to scale the application to keep up with user demand.</p>
             <p>If you haven't worked through the earlier sections, start from <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/">Using minikube to create a cluster</a>.</p>
 
             <p><em>Scaling</em> is accomplished by changing the number of replicas in a Deployment</p>
```
Comment (dipesh-rawat) [inline content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html:32]: Very minor _nit_ in the existing content.
```suggestion
            <p><em>Scaling</em> is accomplished by changing the number of replicas in a Deployment.</p>
```

File: content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 2]
```diff
@@ -49,6 +45,14 @@ <h3>Summary:</h3>
         </div>
         <br>
```
Comment (dipesh-rawat) [inline content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html:46]: Could we possibly eliminate this `<br>` tag to minimize extra spacing above the note?
```suggestion
```

**Review by dipesh-rawat — COMMENTED** [review]
Overall the change looks good. Thanks!

/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
/lgtm

**Comment by reylejano** [comment]
Visually, the page is improved with this PR
/approve

**Pull request merged**